fullstop-jobs/src/main/java/org/zalando/stups/fullstop/jobs/common/HttpCallResult.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (C) 2015 Zalando SE (http://tech.zalando.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zalando.stups.fullstop.jobs.common;
+
+public class HttpCallResult {
+    private  boolean isSecured;
+    private String message;
+
+
+    public HttpCallResult(boolean isSecured, String message) {
+        this.isSecured = isSecured;
```

Comment:
I would not call something "secure", just because we did not find a violation. We could have missed something. It is easier to say something is insecure, in case we find any problem.